src/Compilers/CSharp/Test/IOperation/IOperation/IOperationTests_ISwitchExpression.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Roslyn.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests
+{
+    public partial class IOperationTests_Patterns : SemanticModelTestBase
+    {
+        [CompilerTrait(CompilerFeature.IOperation, CompilerFeature.Patterns)]
+        [Fact]
+        public void SwitchExpression_Basic()
```

Comment:
>SwitchExpression_Basic [](start = 20, length = 22)

We should at least cover basic scenarios of different pieces present and missing, Guard, locals, empty set of arms, missing either side of ```=>``` etc.